src/pages/home/base/assurance/tabs/index.md

```diff
@@ -2,10 +2,20 @@
 noIndex: true
 ---
 
-<Variant platform="android" task="import-library" repeat="5"/>
+<Variant platform="android" task="import-library" repeat="6"/>
 
 1. Add the following libraries in your project's `build.gradle` file:
```

Comment:
We should move this under Groovy and add a similar line for build.gradle.kts for kotlin - repeat count will need to be bumped as well.
Alternatively, you can use a more generic text in here like for the other extensions: `... using the app's Gradle file.`